.github/workflows/images.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
           platforms: linux/amd64,linux/arm64,linux/s390x
```

Comment:
Not part of your change, Is there a reason we don't have `linux/ppc64le` for executor-debug?